packages/manager/modules/pci/src/projects/new/components/pci-new-project-promotion-banner/controller.js

```diff
@@ -0,0 +1,9 @@
+export default class PciNewProjectPromotionBannerController {
+  isPromotionBannerDisplayed() {
+    return this.model && this.model.voucher && this.model.voucher.valid;
```

Comment:
**nitpick:** Can be combined into 1 statement
```suggestion
    return this.model?.voucher?.valid;
```

Reply:
Done.